ci/do.sh

```diff
@@ -150,21 +141,41 @@ fi
 
 if [ "x${FIRST_TIME_BUILD}" == "x1" -a "x${IS_CI_BUILD}" == "x1" ]; then
   echo "### First time build is complete ###"
+  host=$(hostname -s)
+  echo "READY:${host}" >>${config_dir}/.status
   exit 0
 fi
 
+if [ "x${ID}" == "xcentos" -a "x${VERSION_ID}" == "x8" ]; then
+  if [ ! -f /tmp/swig/swig/configure ]; then
+    # source install swig
+    dnf -y install gcc-c++ byacc pcre-devel
```

Comment:
shouldn't we check if dnf was successful or not?

Reply:
The do.sh script has a shebang line of "!/bin/bash -ex" Thus it exits with appropriate exit codes if any command fails.